File: Python/Kode Program/Kartu.py
class Kartu:
    def __init__(self, namaPermainan="", bahan=""):
        self.namaPermainan = namaPermainan
        self.bahan = bahan

    def setNamaPermainan(self, np): self.namaPermainan = np
    def setBahan(self, b): self.bahan = b

    def getNamaPermainan(self): return self.namaPermainan
    def getBahan(self): return self.bahan

    def printInfo(self):
        print(f"Permainan: {self.namaPermainan}, Bahan: {self.bahan}")

File: Java/Kode Program/Kartu.java
public class Kartu {
    protected String namaPermainan;
    protected String bahan;

    public Kartu() {}

    public Kartu(String namaPermainan, String bahan) {
        this.namaPermainan = namaPermainan;
        this.bahan = bahan;
    }

    public void setNamaPermainan(String np) { this.namaPermainan = np; }
    public void setBahan(String b) { this.bahan = b; }

    public String getNamaPermainan() { return namaPermainan; }
    public String getBahan() { return bahan; }

    public void printInfo() {
        System.out.println("Permainan: " + namaPermainan + ", Bahan: " + bahan);
    }
}
